apps/11gist-blog/script.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    $.ajax('https://api.github.com/users/clucio72/gists', {
+        success: function(gists) {
+            gists.forEach(function(gist) {
+                if (gist.description.indexOf('#post Here is my post') > -1) {
+                    var $a = $('<a href="' + gist.url + '">' + gist.description.slice(6) + '</a>');
```

Comment:
OOOO why 6? This is what we call a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

Not a big deal, but something to learn!